lib/huginn_scheduler.rb

```diff
@@ -97,22 +98,23 @@ class HuginnScheduler < LongRunnable::Worker
 
   FAILED_JOBS_TO_KEEP = 100
   SCHEDULE_TO_CRON = {
-    '1m'  => '*/1 * * * *',
-    '2m'  => '*/2 * * * *',
-    '5m'  => '*/5 * * * *',
+    '1m' => '*/1 * * * *',
+    '2m' => '*/2 * * * *',
+    '5m' => '*/5 * * * *',
     '10m' => '*/10 * * * *',
     '30m' => '*/30 * * * *',
-    '1h'  => '0 * * * *',
-    '2h'  => '0 */2 * * *',
-    '5h'  => '0 */5 * * *',
+    '1h' => '0 * * * *',
+    '2h' => '0 */2 * * *',
+    '5h' => '0 */5 * * *',
     '12h' => '0 */12 * * *',
-    '1d'  => '0 0 * * *',
-    '2d'  => '0 0 */2 * *',
-    '7d'  => '0 0 * * 1',
+    '1d' => '0 0 * * *',
+    '2d' => '0 0 */2 * *',
+    '7d' => '0 0 * * 1'
   }
 
   def setup
-    tzinfo_friendly_timezone = ActiveSupport::TimeZone::MAPPING[ENV['TIMEZONE'].presence || "Pacific Time (US & Canada)"]
+    Time.zone = ENV['TIMEZONE'].presence || 'Pacific Time (US & Canada)'
+    tzinfo_friendly_timezone = Time.zone.tzinfo.identifier
```

Comment:
This looks correct.  ActiveSupport::TimeZone::MAPPING should not be a public interface to get the timezone identifier.